webapp/src/store/index.js

```diff
@@ -207,10 +207,20 @@ export default new Vuex.Store({
 			if (state.activeRoom.id !== room) return
 			state.reactions = reactions
 		},
-		'api::world.updated' ({ state, commit }, { world, rooms, permissions }) {
+		'api::world.updated' ({ state, dispatch, commit }, { world, rooms, permissions }) {
+			const oldCurrentRoomModules = state.rooms.find(r => r.id === state.activeRoom.id).modules.map(m => m.type)
 			state.world = world
 			state.permission = permissions
 			commit('updateRooms', rooms)
+			const currentRoom = state.rooms.find(r => r.id === state.activeRoom.id)
```

Comment:
Same here, it feels like this should just work:
```suggestion
			const currentRoom = state.activeRoom
```